### DISCUSSION TITLE
[Question][Hw4] json_parser pow

### DISCUSSION THREAD

**Discussion by polish23** [prosyslab-classroom/cs348-information-security#554] [Q&A]
![image](https://user-images.githubusercontent.com/79696492/117918029-45bb5980-b325-11eb-844d-c74635fc097f.png)

After fixing the bug with FDiv, a new bug has occured.

I didn't revised anything in json_parser.c

How can I fix it?

Thanks.

**Comment by KihongHeo** [comment]
Please use the Makefile.
If you want to do manually, you can follow the instruction. I.e., `-lm` is needed.
